genesis/utils/array_class.py

```diff
@@ -2004,6 +2005,51 @@ def __init__(self):
         return ClassEntitiesState()
 
 
+# =========================================== StaticRigidSimConfig ===========================================
+
+cache_value = partial(dataclasses.field, metadata={FIELD_METADATA_CACHE_VALUE: True})
+
+
+@dataclasses.dataclass
+class StaticRigidSimDummy:
```

Comment:
Why do we have both `StaticRigidSimConfig` and `StaticRigidSimDummy`. We should get rid of the former?

Besides, the name is not quite right. `StaticRigidSimDummy` does not mean anything. What is this class all about? If it is used for kernel caching disambiguation only, then it should rather be called `RigidSimMetadata`, or `RigidSimMetadataCache`. But `dummy` here is just misleading I think.

Reply:
renamed to StaticRigidSimCacheKey/static_rigid_sim_cache_key

Reply:
Yes, whatever but not dummy :)